.github/workflows/integration-tests.yml

```diff
@@ -5,11 +5,21 @@ on:
     branches: [ "main" ]
   push:
     branches: [ "main" ]
+  workflow_dispatch:
```

Comment:
i think the intent is to have test-plugin.yml be the dispatch point... what's this change for?

Reply:
I think @NathanielMcVicar  might have suggested doing that for me to be able to manually trigger the workflows #71 but I could have misconstruded that

Reply:
That's right, but since it seems like you may not have the permissions it probably doesn't help much. Regardless seems worth allowing for people making changes to these in general (or wanting to test it in their own forks where the permission issues won't exist).

Reply:
my point is this script just dispatches to the other script, why not just use that one?